### ISSUE TITLE
Add issue support configuration for OpenStreetMap on translatewiki.net

### ISSUE THREAD

**Issue by wangombe-g** [openstreetmap/openstreetmap-website#4546]
### Problem

At the moment, https://translatewiki.net/w/i.php?title=Translating_talk:OpenStreetMap is configured as the support page for OSM translation string issues on https://translatewiki.net. However, this page seems not to receive attention.

### Description

CC: @translatewiki follow-up for phabricator task: [T357386](https://phabricator.wikimedia.org/T357386)

I work with the Language Team at Wikimedia Foundation and we would like to suggest a template be configured on Github issues for translation message issues that may occur during localization efforts emanating from https://translatewiki.net. As mentioned in the problem statement, these issue are currently [going unaddressed](https://translatewiki.net/w/i.php?title=Translating_talk:OpenStreetMap) which may impact the quality of localized strings for OSM.

Perhaps posting these translation message issue on Github would be ideal for visibility. At the moment, we have an [open PR (patch)](https://gerrit.wikimedia.org/r/c/translatewiki/+/1006615/1/groups/OpenStreetMap/OpenStreetMap.yaml) on Gerrit which basically proposes adding translation message issues to Github using the `bug` and `i18n` label. This may not be the most optimal solution. An issue template would be ideal.

**Comment by gravitystorm** [comment]
I think the key issue is to make sure that as much of the translation queries that can be dealt with by our community at translatewiki are dealt with there. For example, if there's questions like "what should the spanish translation be for 'Relation'" or "I don't agree with this translation in German" then that's something that should be asked and answered within the translatewiki community, since here we are mostly programmers who don't have any good answers for that kind of thing. Very few, if any, of the translators are active here on this repo, so I think there's even less chance for many of the queries to be answered.

If there's a situation were something needs to change in our codebase to help support translators (e.g. [avoid using `zero:` key](https://github.com/openstreetmap/openstreetmap-website/pull/4077)) then that's definitely something for here.

I've looked through the linked page, and it would be very hard for us to deal with many of the comments there. (I'm also not sure how to reply to most of those, even if I wanted to, since some parts of the page have "Reply" links and others don't, but that's a different problem).

**Comment by wangombe-g** [comment]
I can acknowledge that there are language specific issues on the Translatewiki OSM support page that should be answered by the translation community. Issues with the locale strings, however, are the responsibility of the developer and not the translations community. For instance, the developer is responsible for addressing [this issue](https://translatewiki.net/wiki/Osm:Site.copyright.legal_babble.contributors_nl_and/es) where there is a broken link by updating [en.yml:L2105](contributors_nl_and) to an existing link. 

The green box represents the language the translator was translating into.

![image](https://github.com/openstreetmap/openstreetmap-website/assets/12006290/30c43855-f7d2-475b-9600-c7ccdbb9f2bf)

Translslatewiki, using the yml file format which OSM is configured to use for localization, typically represents nesting using a period, i.e ".". For example, in the screenshot above, the string in question is located in the file [en.yml:L3103](tracestracktop_top): where the strings follow a hierarchy in OSM like so:

|   |   |   |   |
|---|---|---|---|
| javascripts|   |   |   | 
|   | map |   |   | 
|   |   | base |   |
|   |   |    | tracestracktop_top | 

![image](https://github.com/openstreetmap/openstreetmap-website/assets/12006290/bb7599e0-4256-472a-8bf9-4f0297d9afdb)
 
In this screenshot, the translator is asking that the word 'doesn't' located here [en.yml:L2418](https://github.com/openstreetmap/openstreetmap-website/blob/master/config/locales/en.yml#L2418), be considered for revision as it may not translate well to Turkish or other languages.

Here is a link to [Translatewiki message documentation ](https://www.mediawiki.org/wiki/Help:System_message/en#Message_documentation) which describes how translation source strings should be structured and how describing them can aid translators in understating context. A majority of these issues are also in regard to context.

**Comment by Nikerabbit** [comment]
In case you would get language specific questions here, we can see how we can improve the user interface in our side to better redirect language specific topics to our language portals.

**Comment by Nikerabbit** [comment]
Circling back to this, do you feel comfortable for making a decision to unblock us? For example:

* Yes, we can direct questions to here and monitor whether the questions are in scope. If too many are out of scope, revert the change until further improvements are made
* No, keep pointing questions on the talk page (and have someone watch that page to comment on relevant questions)

**Comment by tordans** [comment]
From what I understand, one issue are non-ideal translation keys that are specified inside the code. I am thinking about cases like https://github.com/openstreetmap/openstreetmap-website/pull/4602/files and …

> In this screenshot, the translator is asking that the word 'doesn't' located here [en.yml:L2418](https://github.com/openstreetmap/openstreetmap-website/blob/master/config/locales/en.yml#L2418), be considered for revision as it may not translate well to Turkish or other languages.

For those cases, we could create a specific issue template ([code example](https://github.com/openstreetmap/openstreetmap-website/blob/master/.github/ISSUE_TEMPLATE/feature.yml?plain=1)) like the first two on https://github.com/openstreetmap/openstreetmap-website/issues/new/choose.

Ideally, translators would have docs somewhere in translatewiki that explain this specific use case and from there link directly into the issue template.

**Comment by wangombe-g** [comment]
I think this approach is viable; that is, creating an issue template for translation source strings.

**Comment by wangombe-g** [comment]
Hi @tordans, it's not clear which option above you reacted to with the 👍.

**Comment by wangombe-g** [comment]
This [solution was implemented](https://phabricator.wikimedia.org/T357386#9799227) as a placeholder up until a suitable issue template for translation source strings is provided.
